Payment methods: Implemented payments page to manage saved cards and bank accounts

udaykiranchimpiri opened the pull request:
Added a new Payments page under Account Settings to manage payment methods.  
Currently supports saving cards and bank accounts.

chimpiriudaykiran committed:
Payment Methods: Implemented payment methods page to manage saved cards and bank accounts

- Added a new Payments page under Account Settings to manage payment methods.
- Currently supports saving cards and bank accounts.

weic6 reviewed (commented):
[Edited] i will review this PR later.

weic6 reviewed (commented):
cannot add paypal account for now, hope we can fix it after the backend paypal apis are done

weic6 commented:
adding card: if unchecking the box "Billing info is different from shipping" , it says "Error: Required property". maybe make the box checked by default? or make the error message clearer like "add add your shipping address"
<img width="1016" height="589" alt="image" src="https://github.com/user-attachments/assets/e491c2ce-b4ba-4f1e-bad2-1ac356ddfcb3" />

for editing card: 
<img width="974" height="782" alt="image" src="https://github.com/user-attachments/assets/0e0a72c5-239e-4f84-8edc-dec6ec2c7163" />
after clicking, it says "Error Unauthorized: No token provided"
<img width="993" height="785" alt="image" src="https://github.com/user-attachments/assets/965b03a9-e87a-4731-a753-c217dca94bb3" />

chimpiriudaykiran committed:
Made the billing address default and return error if billing address is empty

udaykiranchimpiri committed:
Made the billing address default and return error if billing address is empty

Updated logic to make billing address a required field during payment method submission.
Added validation to return an error if billing address is missing.

udaykiranchimpiri committed:
Merge branch 'payment-methods-udaykiran' of https://github.com/The-Donovans-Venom-501c3/the-donovans-piano-room into payment-methods-udaykiran
